content/blog/2025/06/kubewarden-1.26-release.md

```diff
@@ -0,0 +1,214 @@
+        registry://ghcr.io/kubewarden/policies/ingress:v0.1.8
+```
+
+Thanks to suggestions from our user community, `kwctl` now can consume a YAML
+file containing the Custom Resource Definition of policies, and run the request
+against them. Using this run mode, `kwctl` evaluates not only 1 policy, but
+each policy in the file using the same request. This works for all types of
+policies, including PolicyGroups.
+
+For example, with the following definition of 2 policies:
+
+```yaml
+# policies.yaml
+---
+apiVersion: policies.kubewarden.io/v1
+kind: ClusterAdmissionPolicy
+metadata:
+  name: safe-labels
+spec:
+  module: registry://ghcr.io/kubewarden/policies/safe-labels:v1.0.2
+  rules:
+    - apiGroups: [""]
+      apiVersions: ["v1"]
+      operations: ["CREATE"]
+      resources: ["namespaces"]
+  settings:
+    denied_labels:
+      - cost-center
+---
+apiVersion: policies.kubewarden.io/v1
+kind: ClusterAdmissionPolicy
+metadata:
+  name: capabilities-psp
+spec:
+  module: registry://ghcr.io/kubewarden/policies/capabilities-psp:v1.0.2
+  rules:
+    - apiGroups: [""]
+      apiVersions: ["v1"]
+      operations: ["CREATE", "UPDATE"]
+      resources: ["pods"]
+  settings:
+    allowed_capabilities:
+      - CHOWN
+    required_drop_capabilities:
+      - NET_ADMIN
+```
+
+We can run a request locally with kwctl:
+
+```console
+$ kwctl run policies.yaml --request-path req_pod_with_allowed_capabilities_accept.json
+
+  Successfully pulled policy from registry://ghcr.io/kubewarden/policies/safe-labels:v1.0.2
+  Successfully pulled policy from registry://ghcr.io/kubewarden/policies/capabilities-psp:v1.0.2                                                                                                                                                                                                                                2025-06-25T12:25:57.144579Z  WARN kwctl::command::run: Multiple policies defined inside of the CRD file. All of them will run sequentially using the same request.
+{"uid":"f0b8fba3-4f4f-465b-af8c-84d0326a2dc2","allowed":true,"auditAnnotations":null,"warnings":null}
+{"uid":"f0b8fba3-4f4f-465b-af8c-84d0326a2dc2","allowed":true,"patchType":"JSONPatch","patch":"W3sib3AiOiJhZGQiLCJwYXRoIjoiL2FwaVZlcnNpb24iLCJ2YWx1ZSI6InYxIn0seyJvcCI6ImFkZCIsInBhdGgiOiIva2luZCIsInZhbHVlIjoiUG9kIn0seyJvcCI6ImFkZCIsInBhdGgiOiIvc3BlYy9jb250YWluZXJzLzAvc2VjdXJpdHlDb250ZXh0L2NhcGFiaWxpdGllcy9kcm9wIiwidmFsdWUiOlsiTkVUX0FETUlOIl19XQ==","auditAnnotations":null,"warnings":null}
+```
+
+In this run mode, only the following attributes of the Custom Resource
+Definition (CRD) are evaluated:
+
+- Policy module
+- Policy settings
+- Context-aware resources the policy can access
+
+Other fields, such as rules, matchConditions, objectSelector, and
+namespaceSelector, are ignored, as they cannot be applied out of cluster.
+
+You can read about this by performing a `kwctl run --help` or by reading our
```

Comment:
```suggestion
Moreover, it's now possible to run ["policy groups"](https://docs.kubewarden.io/explanations/policy-groups) policies using kwctl! 🥳

You can read about this by performing a `kwctl run --help` or by reading our
```

Reply:
This is also on line 27, but it indeed deserves its own paragraph.

Reply:
Done on the starting paragraphs with https://github.com/kubewarden/kubewarden.io/pull/350/commits/0c9f284967a3025707fbd3d34791d2ef651f5cb6.